pkg/sinit/init.go

```diff
@@ -245,8 +254,14 @@ func InitializeLogging(ctx context.Context, config LoggingConfig) error {
 
 // Flush flushes all pending logs to the configured outputs. Blocks until all logs are written.
 func Flush() {
-	handlerSplunk.Close()
-	handlerCloudWatch.Close()
+	if handlerSplunk != nil {
+		handlerSplunk.Close()
+	}
+
+	if handlerCloudWatch != nil {
+		handlerCloudWatch.Close()
+	}
+
```

Comment:
Found a small but - when panicking, defer is still called and these values were all `nil` so panicking as well.